js/components/Search/index.js

```diff
@@ -0,0 +1,5 @@
+import Search from "./Search";
+import SearchContainer from "./SearchContainer";
```

Comment:
Components do not have containers. Please make changes to your imports and exports.